ap_versioned_docs/version-2.10.0/admin-guide/custody-services/configuration.mdx

```diff
@@ -0,0 +1,130 @@
+---
+title: Configuration
+sidebar_position: 10
+---
+
+import { CodeExample } from "@site/src/components/CodeExample";
+
+## Custody Server Configuration
+
+If you want to use an external custody service to store and manage your wallets, then you need to deploy one more service - the Custody Server.
+
+This service also needs the `STELLAR_ANCHOR_CONFIG` that was previously mentioned. By default, the `anchor-config-default-values.yaml` config file will be used.
+
+Also, now you don't need to deploy the Stellar Observer since the Custody Server will be responsible for its functionality.
+
+Update the configuration file of the Anchor Platform with the base URL and port.
+
+<CodeExample>
+
+```yaml
+custody_server:
+  # The listening port of the Custody Server.
+  # Default value: 8086
+  port: 8086
+  # The base URL of the Custody Server.
+  # Default value: http://localhost:8086
+  base_url: http://localhost:8086
+```
+
+</CodeExample>
+
+Configure authentication type.
+
+<CodeExample>
+
+```yaml
+custody_server:
+  auth:
+    # Type of authentication that is used when the Anchor Platform communicates with the Custody Server.
+    # Supported values: [none, api_key, jwt]
+    # Default value: none
+    type: none
+```
+
+</CodeExample>
+
+If you set the `api_key` or `jwt` authentication type, then you need to add an environment variable.
+
+<CodeExample>
+
+```bash
+# dev.env
+SECRET_CUSTODY_SERVER_AUTH_SECRET="Custody Server auth secret"
+```
+
+</CodeExample>
+
+Start the Custody Server using Gradle or Docker.
+
+<CodeExample>
+
+```bash
+./gradlew service-runner:bootRun --args=--custody-server
+docker run stellar/anchor-platform:2.9.0 --custody-server
```

Comment:
Should this be `2.10.0`? There are a few other places in this PR that reference 2.9.0.

Reply:
fixed